src/EntityFramework/Core/Objects/ELinq/LinqExpressionNormalizer.cs

```diff
@@ -132,6 +159,51 @@ private static bool IsConstantZero(Expression expression)
         [SuppressMessage("Microsoft.Maintainability", "CA1502:AvoidExcessiveComplexity")]
         internal override Expression VisitMethodCall(MethodCallExpression m)
         {
+#if NETSTANDARD2_1
+            // .NET 10 made changes to overload resolution to prefer Span-based overloads when those exist ("first-class spans").
+            // Unfortunately, the LINQ interpreter does not support ref structs, so we rewrite e.g. MemoryExtensions.Contains to
```

Comment:
Just to be pedantic, I'm not sure that EF6 uses the LINQ interpreter (it might - I'm really not sure). The reason to rewrite is more that EF6 simply doesn't recognize MemoryExtensions.Contains in translation, I think. But at most this means tweaking the comment here, nothing more.